CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project `6.2.x` per each release will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [v6.2.0.8] - 2020-11-10
+
+### Added
+- Add git release tag as a label (refer to [issue](https://github.com/wso2/docker-ei/issues/218))
+- Add ca-certificates
+- Create wso2 user and group
```

Comment:
@NishikaDeSilva shall we please create two issues mapping to these two changes since they are considerable changes?

Reply:
+1